migrations/01-create-artist-table.sql

```diff
@@ -0,0 +1,6 @@
+
+  CREATE TABLE Artists (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  genre VARCHAR(255) NOT NULL
+  );
```

Comment:
This should be formatted so everything inside the statement is indented by one step:
```
CREATE TABLE Artists (
  id SERIAL PRIMARY KEY,
  name VARCHAR(255) NOT NULL,
  genre VARCHAR(255) NOT NULL
);
```